_episodes/06-debugging.md

```diff
@@ -300,20 +300,28 @@ esmvalcore._task.DiagnosticError: Cannot execute script 'diag_scripts/examples/d
 
 The script path should be relative to ``diag_scripts`` directory. It means that
 the script ``diagnostic_timeseries.py`` is located in
-``path_to_esmvaltool/diag_scripts/ocean/``. Alternatively, the script path can
-be an absolute path. To examine this, we change the script path and run the
-recipe:
+``<path_to_esmvaltool>/diag_scripts/ocean/``. Alternatively, the script path can
+be an absolute path. To examine this, we create a small python script `hello.py`:
+
+```python
+with open('hello_world.txt', 'w') as f:
+    print('Fello from ESMValTool!', file=f)
+```
+
+Change and run the recipe:
 
 ```yaml
 40        scripts:
 41          timeseries_diag:
-42            script: path_to_esmvaltool/diag_scripts/ocean/diagnostic_timeseries.py
+42            script: ~/esmvaltool_tutorial/hello.py
 ```
 
 ~~~bash
   esmvaltool run recipe_example.yml
 ~~~
 
+Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own. Have a look through the scripts available in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/diag_scripts/).
+
```

Comment:
```suggestion
Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own.

```

Reply:
"creating your own diagnostic" is another episode in the advanced part of the tutorial that would be developed in the near future. Also, "Available recipe and diagnostic scripts" are already mentioned in the next section.